src/bioregistry/data/bioregistry.json

```diff
@@ -40707,4 +40713,4 @@
       "prefix": "ZP"
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
Normally I would agree but this file is managed by a python script so it gets dumped each time with no newline at the end

Reply:
Fixed in 324dc2c

Reply:
I don't even know how to solve this with my editor (`kate`).

Reply:
(but I'll make care to use `git add -p` next time)